### PR TITLE
Add Jasmin channel type with it's idiot encoding

### PR DESCRIPTION
**Pull request by nicpottier**
One of our aggregators uses Jasmin as their backend (http://www.jasminsms.com/) which at first glance looks nice but does absolutely nothing for you, including encoding or decoding from GSM7 charsets.

We had previously integrated with them using the external channel API, but that doesn't work because we need to en/decode properly.

So it gets its own channel now.

### CODE REVIEW

File: static/less/style.less [hunk 1]
```diff
@@ -2083,7 +2089,7 @@ input[type='radio'] {
   background-color: @color-alert-error + #222;
   border: 1px solid @color-alert-error;
   text-shadow: none;
-  width:520px;
+  width:560px;
```
Comment (nicpottier) [inline static/less/style.less]: This fixes errors wrapping on horizontal forms.

File: temba/channels/models.py [hunk 2]
```diff
@@ -963,6 +968,70 @@ def build_send_url(cls, url, variables):
         return url
 
     @classmethod
+    def send_jasmin_message(cls, channel, msg, text):
+        from temba.msgs.models import Msg, WIRED
+        from temba.utils import gsm7
+
+        # build our callback dlr url, jasmin will call this when our message is sent or delivered
+        dlr_url = 'https://%s%s' % (settings.HOSTNAME, reverse('handlers.kannel_handler', args=['status', channel.uuid]))
```
Comment (ericnewcomer) [inline temba/channels/models.py]: kannel?

File: templates/channels/channel_claim.haml [hunk 3]
```diff
@@ -336,6 +336,21 @@
                 Connect your <a href="http://www.kannel.org/" target="_blank">Kannel</a> instance, we'll walk you through
```
Comment (ericnewcomer) [inline templates/channels/channel_claim.haml]: oops

Comment (ericnewcomer) [inline templates/channels/channel_claim.haml]: nm

**Comment by ericnewcomer** [comment]
yep!

**Pull request merged**